test/rules/consecutive-function-overloads/test.ts.lint

```diff
@@ -0,0 +1,63 @@
+// good
+
+interface i1 {
+    a();
+    a(x: number);
+    b();
+    b(x: string);
+}
+
+interface i2 {
+    a();
+    a(x: number);
+    a(): void;
+    b();
+    b(x: string);
+}
+
+interface i3 {
+    a();
+    "a"();
+}
+
+interface i4 {
+    a();
+    ["a"]();
+}
+
+
+// bad
```

Comment:
Good catch! I don't think this code will work for nested interfaces right now since it's only sweeps interfaceDeclarations. I probably would have to check if an interface member is an object and then recurse for them. Do you have any other suggestion?